cves/kernel/CVE-2016-9685.yml

```diff
@@ -129,10 +130,12 @@ unit_tested:
 
     For the fix_answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
-  code:
-  code_answer:
-  fix:
-  fix_answer:
+  code: false
+  code_answer: |
+    There were no unit tests for the fs subsystem
```

Comment:
This surprises me - are you sure?

Reply:
I'll take another look. I have limited knowledge of the linux kernel and based on my research it seemed unit testing libraries were kept separate from the kernel itself